packages/dds/tree/src/feature-libraries/default-schema/defaultFieldKinds.ts

```diff
@@ -171,8 +173,38 @@ export const forbidden = new FieldKindWithEditor(
 	new Set(),
 );
 
+const fieldKindConfigurations: ReadonlyMap<number, FieldKindConfiguration> = new Map([
+	[
+		0,
+		new Map<FieldKindIdentifier, FieldKindConfigurationEntry>([
+			[nodeKey.identifier, { kind: nodeKey, formatVersion: 0 }],
+			[required.identifier, { kind: required, formatVersion: 0 }],
+			[optional.identifier, { kind: optional, formatVersion: 0 }],
+			[sequence.identifier, { kind: sequence, formatVersion: 0 }],
+			[forbidden.identifier, { kind: forbidden, formatVersion: 0 }],
+		]),
+	],
+	[
```

Comment:
this bit should go away in this PR and be merged alongside a full set of changes that enable optional/required field new versions.